bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -193,7 +245,7 @@ public void run() {
                     isr.close();
                 } catch (IOException e) {
                     logger.error("An exception occurred while reading the stdout when executing '{}' : '{}'",
```

Comment:
And this should be `warn`, also below. `error` is reserved for the framework of occasions where the stability of the system may be affected.